test/document/reducer.test.ts

```diff
@@ -1,5 +1,5 @@
 import { Action, CreateChildDocumentInput, utils } from '../../src/document';
-import { setName } from '../../src/document/actions';
+import { setName } from '../../src/document/actions/creators';
```

Comment:
this also caused a problem with vitest (see above comment). importing directly fixes it